sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubWebJobsBuilderExtensions.cs

```diff
@@ -96,22 +96,22 @@ internal static void ConfigureInitialOffsetOptions(EventHubOptions options)
                         options.EventProcessorOptions.DefaultStartingPosition = EventPosition.Latest;
                         break;
                     case OffsetType.FromEnqueuedTime:
-                        try
+                        if (!options.InitialOffsetOptions.EnqueuedTimeUtc.HasValue)
                         {
-                            options.EventProcessorOptions.DefaultStartingPosition = EventPosition.FromEnqueuedTime(options.InitialOffsetOptions.EnqueuedTimeUtc.Value);
-                        }
-                        catch (FormatException fe)
-                        {
-                            string message = $"{nameof(EventHubOptions)}:{nameof(InitialOffsetOptions)}:{nameof(InitialOffsetOptions.EnqueuedTimeUtc)} is configured with an invalid format. " +
-                                "Please use a format supported by DateTime.Parse().  e.g. 'yyyy-MM-ddTHH:mm:ssZ'";
-                            throw new InvalidOperationException(message, fe);
+                            throw new InvalidOperationException(
+                                "A time must be specified for 'enqueuedTimeUtc', when " +
+                                "'initialOffsetOptions.type' is set to 'fromEnqueuedTime'.");
```

Comment:
Well we had actual properties before and still elected to not use the names, so I assumed that was because camelcasing matches our docs for these host.json files.